tests/sgl/device/test_texture_loader.cpp

```
@@ -0,0 +1,87 @@
+                    CHECK(std::memcmp(path_row, stream_row, packed_layout.row_pitch) == 0);
+                }
+            }
+        }
+    }
+}
+
+} // namespace
+
+TEST_CASE_GPU("texture_loader_stream_bitmap")
+{
+    const std::filesystem::path path = platform::project_directory() / "data" / "test_images" / "albert.jpg";
```

Cppcheck (2.21.0)

[style] The function 'enum_has_value' is never used.

(unusedFunction)